src/main/include/utils/CommandUtils.h

```diff
@@ -29,46 +29,47 @@ static frc2::CommandPtr OutakeUntilTopNotPresent(IntakeSubsystem* intake) {
       .FinallyDo([intake] { intake->Stop(); });
 }
 
-static frc2::CommandPtr Score(std::function<ScoringDirection()> direction,
-                              ScoringSubsystem* scoring,
-                              IntakeSubsystem* intake) {
-  return (
-    // (NoteShuffle(intake).ToPtr())
-    //           .AlongWith(frc2::InstantCommand([scoring] {
-    //                        scoring->SpinOutake();
-    //                      }).ToPtr())
-    //           .WithTimeout(2_s)
-              // .AndThen(frc2::InstantCommand([] {
-              //            ConsoleLogger::getInstance().logVerbose(
-              //                "Scoring Composition", "shuffled %s", "");
-              //          }).ToPtr())
-              // .AndThen(frc2::WaitCommand(0.4_s).ToPtr())
-              // .AndThen(
-                FlywheelRamp(intake, scoring, direction).ToPtr()
-                // )
+static frc2::CommandPtr ScoreRamp(std::function<ScoringDirection()> direction,
+                             ScoringSubsystem* scoring,
+                             IntakeSubsystem* intake) {
+  return (FlywheelRamp(intake, scoring, direction)
+              .ToPtr()
               .AndThen(frc2::InstantCommand([] {
                          ConsoleLogger::getInstance().logVerbose(
                              "Scoring Composition", "flywheel ramped %s", "");
-                       }).ToPtr())
-              .AndThen(frc2::WaitCommand(kFlywheelRampDelay).ToPtr())
-              .AndThen(
-                  Feed(intake, scoring, direction).ToPtr().WithTimeout(2_s))
-              .AndThen(frc2::InstantCommand([] {
-                         ConsoleLogger::getInstance().logVerbose(
-                             "Scoring Composition", "fed %s", "");
-                       }).ToPtr())
+                       }).ToPtr()));
```

Comment:
Need to add a timeout and an unless to check for the presence of a note first

Reply:
Should also have a FinallyDo that stops the motors but only runs if there's a timeout

Reply:
how long should the timeout be?